packages/forms-web-app/src/dynamic-forms/dynamic-components/multi-file-upload/question.js

```diff
@@ -107,16 +106,12 @@ class MultiFileUploadQuestion extends Question {
 	 * @returns {Promise<void>}
 	 */
 	async saveAction(req, res, journey, section, journeyResponse) {
-		// check for validation errors
-		const errorViewModel = this.checkForValidationErrors(req, section, journey);
-		if (errorViewModel) {
-			return this.renderAction(res, errorViewModel);
-		}
```

Comment:
This code boggled my mind. It looks like it should do something except that `this.checkForValidationErrors` (as you'll see in a few lines) just returns void.